Git tagging

HN23 opened the pull request.

HN23 commented:
example of tag created:
![image](https://user-images.githubusercontent.com/3018905/203351885-ad864d37-bed2-4c79-a5f7-862edd80d545.png)

Btodhunter reviewed (approved):
I think this LGTM.... Would love to see this whole workflow in action before all the pieces get merged though.

HN23 commented:
> I think this LGTM.... Would love to see this whole workflow in action before all the pieces get merged though.

Hey @Btodhunter thanks for the approval. Heres an example workflow i did with the 'release-management-dashboard'

https://app.circleci.com/pipelines/github/anchore/release-management-dashboard/21/workflows/abade118-6c4f-46c6-bf5d-c8dce534338e/jobs/23

from the circleci, you can see a tag was created:
https://app.circleci.com/pipelines/github/anchore/release-management-dashboard

I've deleted the tags and rebased/cleaned up the commits on the project already, but you can trigger the pipeline again if you wanna see it run through.

HN23 commented on anchore-ci/release_tasks:
```diff
@@ -482,4 +482,42 @@ anchore-check-image-by-id() {
+
+    git pull
+    git checkout ${GIT_TAG_SOURCE}
+    git switch -c tempBranchForTagCutting
```
dont need this.
